File: modules/image/src/com/trailmagic/image/HeavyImageManifestation.java
/*
 * Copyright (c) 2004, 2005 Oliver Stewart.  All Rights Reserved.
 */
package com.trailmagic.image;

import java.sql.Blob;

public class HeavyImageManifestation extends ImageManifestation {
    //    private byte[] m_data;
    private Blob m_data;

    public HeavyImageManifestation() {
        super();
    }

    public Blob getData() {
        return m_data;
    }

    public void setData(Blob data) {
        m_data = data;
    }
    /*
    public byte[] getData() {
        return m_data;
    }

    public void setData(byte[] data) {
        m_data = data;
    }
    */

}

File: image/src/main/scala/com/trailmagic/image/HeavyImageManifestation.scala
package com.trailmagic.image

import com.trailmagic.image.security.Identity
import java.sql.Blob
import reflect.BeanProperty

@Identity(classOf[ImageManifestation])
class HeavyImageManifestation() extends ImageManifestation {
  @BeanProperty var data: Blob = null;

  def this(data: Blob) {
    this()
    this.data = data
  }
}
